### PR TITLE
add duration test and owner test

### PR DESCRIPTION
**Pull request by colink14110476**
This PR will add some unit tests to cover some additional bases.
I'll also create a helper contract to easily deploy and set up all contracts to goerli. 
Will likely be on hold until after the admin PR is pushed through.

### CODE REVIEW

**Comment by colink14110476** [comment]
Active Goerli Deployments:
```
Nounders DAO setting to deployer (0x810c1850C9B726D691dF9577B128EDf28829F33e)...
NounsToken contract deployed to 0xC38025c0Be3DEb92Ff66Ab94a5D69c244FaE4922
NounsAuctionHouseProxy contract deployed to 0x7549dF3081A29b11bAf0eDa1F569291f41D85318
NounsDAOExecutor contract deployed to 0xdA15baC0d8e7a736D24A1d973896628CCA55F03D
NounsDAOProxyV2 contract deployed to 0x2cb826fcde8047b68d8bf09C4B4AA0ca89C12F4a
```

File: packages/nouns-contracts/test/auction.test.ts [hunk 1]
```diff
@@ -325,4 +325,23 @@ describe('NounsAuctionHouse', () => {
       .to.emit(nounsAuctionHouse, 'AuctionSettled')
       .withArgs(nounId, '0x0000000000000000000000000000000000000000', 0);
   });
+  it('verify duration change from owner', async () => {
```
Comment (jackdishman) [inline packages/nouns-contracts/test/auction.test.ts]: Should this test replace the current test or add as an additional test?

```
  it('should set the duration of an auction', async () => {
    await (await nounsAuctionHouse.unpause()).wait();
    await (await nounsAuctionHouse.setDuration(60 * 60 * 25)) // sets duration to 25 hours
    let newDuration = await nounsAuctionHouse.duration();
    await expect(newDuration).to.be.equal(60 * 60 * 25)

    await nounsAuctionHouse.connect(executor).setDuration(60 * 60 * 26) // sets duration to 26 hours
    newDuration = await nounsAuctionHouse.duration();
    await expect(newDuration).to.be.equal(60 * 60 * 26)
  });
```

**Review by jackdishman — APPROVED** [review]

File: packages/nouns-contracts/test/auction.test.ts (hunk 1 above)
Comment (colink14110476) [reply packages/nouns-contracts/test/auction.test.ts]: Woops ya I should merge the most recent commit into this one. I'll probably just add an additional test case under yours.

**Comment by colink14110476** [comment]
npx hardhat test output:
```
  124 passing (3m)
  2 pending
```

**Pull request merged**